pkgs/top-level/all-packages.nix

```diff
@@ -34728,4 +34728,6 @@ with pkgs;
   honeytail = callPackage ../servers/tracing/honeycomb/honeytail { };
 
   honeyvent = callPackage ../servers/tracing/honeycomb/honeyvent { };
+
+  s5 = callPackage ../tools/security/s5 { };
```

Comment:
Location seems off. The list is no longer really alphabetically. I see it more near `subfinder`.

Reply:
I moved it up the stack! 👍